### ISSUE TITLE
Insiders version does not display GitHub stars

### ISSUE THREAD

**Issue by fkorotkov** [squidfunk/mkdocs-material#2382]
I'm using the latest insiders build to build Cirrus CI documentation and after building the website it misses the stars.

To reproduce it, you can clone https://github.com/cirruslabs/cirrus-ci-docs and then run the insiders version locally

```console
docker pull ghcr.io/squidfunk/mkdocs-material-insiders:latest
./scripts/start-insiders.sh
```

You'll see that GitHub stars are missing in the right top corner. Then you can run the regular version:

```console
docker pull ghcr.io/squidfunk/mkdocs-material:latest
./scripts/start.sh 
```

And you'll see that GitHub stars will appear back.

**Comment by squidfunk** [comment]
Thanks for reporting. Confirmed. The cirrus-ci website already tells us what's wrong - there's no release tag to fetch:

<img width="970" alt="Bildschirmfoto 2021-03-03 um 13 51 41" src="https://user-images.githubusercontent.com/932156/109808599-9f1d5300-7c27-11eb-80fb-91393523012b.png">

Thus we should consider this case and catch it, but continue with the stars/forks.

**Comment by squidfunk** [comment]
Fixed in Insiders 2.2.1. The release tag will now just be omitted when not present, so the stars/forks are displayed despite that.

**Comment by fkorotkov** [comment]
Can confirm. It's working now! Thanks!
